### DISCUSSION TITLE
Standard_D4as_v6 disk mount issue

### DISCUSSION THREAD

**Discussion by zectorpt** [Azure/AKS#4711] [General]
We have encountered an issue with the error message MountVolume.MountDevice failed for volume: rpc error: code = Internal desc = failed to find disk on lun 0, indicating that the kubelet is unable to find the disk on the specified Logical Unit Number (LUN). This problem appears to be related to the specific VM SKU in use (Standard_D4as_v6).

Doing exactly the same with the SKU Standard_D4as_v5 works fine. Please analyse it or create an exclusion in the documentation.

**Comment by Gideon-Felt** [comment]
Experiencing the same issue a few different times with (Standard_D32ads_v6) node pools.
same manifests applied on V5 worked for me as well.

**Comment by yriveiro** [comment]
I did change to v5 and the issue is gone

**Reply by Gideon-Felt** [reply]
Yeah same
